modules/apigee_edge_teams/src/Plugin/EntityReferenceSelection/TeamMembersSelection.php

```diff
@@ -22,6 +22,7 @@
 
 use Drupal\apigee_edge\Entity\Controller\DeveloperControllerInterface;
 use Drupal\apigee_edge_teams\TeamMembershipManagerInterface;
+use Drupal\Component\Utility\Html;
```

Comment:
Import is never used. Can be removed.

Reply:
Removed statement. Thanks for pointing it out. :wink: